hosting/how-to-host-on-sia.md

```diff
@@ -1,151 +1,239 @@
+
+![](../.gitbook/assets/host-start.jpg)
+
+Click **Add a folder**, then select a location. You can choose a drive \(like D:\\\\\) to use an entire empty drive or partition, or you can pick a specific folder on a drive. Sia will create one large file in the location you select that allocates the amount of storage you choose.
 
-* 10240 is 10 GB
-* 51200 is 50 GB
-* 102400 is 100 GB
-* 1024000 is 1 TB
-* 4096000 is 4 TB
-* 10240000 is 10 TB
-* 102400000 is 100 TB
+![](../.gitbook/assets/host-add-folder.jpg)
+
+Tell Sia how much space you want to allocate to hosting, in megabytes. Entering a value lower than 32 GB will provide a warning since as a host you're graded on having a lot of available storage. Set at least 4 TB if you can. You can type in a value if you want to set a value higher than what the slider allows.
 
 ## Host Settings
 
-![](../.gitbook/assets/hosting-3.png)
+![](../.gitbook/assets/host-settings.jpg)
+
+### Max Duration
 
-{% tabs %}
-{% tab title="Max Duration" %}
 Measured in weeks, Max Duration determines the longest contract you'll accept. The default length during contract creation for renters is 13 weeks, so don't go lower than that or you risk not forming many contracts.
-{% endtab %}
 
-{% tab title="Collateral Per TB/Month" %}
-Collateral is the amount of Siacoins you're willing to lose if you end up not fulfilling your end of the contract, which is to store the renter's data for the contract length while being online.
+This can also be changed in `siac` or using Sia-UI Terminal with the command `host config maxduration <value>`, e.g.
+
+`host config maxduration 26w`
+
+### Collateral per TB/Month
+
+Collateral is the amount of Siacoins you’re willing to lose if you end up not fulfilling your end of the contract, which is to store the renter’s data for the contract length while being online.
 
-Putting up more collateral improves trust by showing you're willing to lose more money than another host. Setting this amount at 2-3 times your storage price is a good idea to maximize your host score while not losing too much money if your host does indeed go down.
-{% endtab %}
+Putting up more collateral improves trust by showing you’re willing to lose more money than another host. Setting this amount at 2-3 times your storage price is a good idea to maximize your host score while not losing too much money if your host does indeed go down.
+
+This can also be changed in `siac` or using Sia-UI Terminal with the command `host config collateral <value>`, e.g.
+
+`host config collateral 100SC`
+
+### Storage per TB/Month
 
-{% tab title="Storage Per TB/Month" %}
 The biggest factor in your host score, this number is how many Siacoins you are charging for 1 TB each month. The less you charge, the more positively it affects your host score. The more you charge, the more money you make. Find your balance between profitability and competitive pricing.
```

Comment:
Probably should mention that some partners have limited their max storage pricing.